### DISCUSSION TITLE
[Question] Can I have an object invisible to one camera but visible to other camera in the same place.

### DISCUSSION THREAD

**Discussion by chandramohanjagadeesan** [isaac-sim/IsaacLab#1056] [Q&A]
I have two objects and two cameras in an environment. I would like the one object to be visible to one camera but not to the other. I have a 3D reconstruction task of an unknown object. I am planning to have an imaginary dome around the object. Based on how much surface area of the imaginary dome is covered by the camera I would like to give rewards. So I need to reconstruct the dome (imaginary) and the object inside the dome (main object). So if I have the above feature it would be greatly helpful. I found that is still not possible from an older  thread. Can someone tell me if the feature is available now and if not can you please open a issue for development?

**Comment by glvov-bdai** [comment]
I do not believe that this is currently possible. If you only have two static objects, you could use a ray caster camera to make each object visible in their respective cameras. (each ray caster can see one static object)

(see ray_caster_visible_mesh_prim_paths arg in https://isaac-sim.github.io/IsaacLab/source/how-to/estimate_how_many_cameras_can_run.html)

**Reply by chandramohanjagadeesan** [reply]
Thanks for the reply. For now my objects are static and I will try to use the ray caster.
